.readthedocs.yaml

```diff
@@ -0,0 +1,26 @@
+# .readthedocs.yaml
```

Comment:
Any chance this file can move inside docs/ to reduce clutter in the root dir?

Reply:
Looks like not 
![image](https://user-images.githubusercontent.com/4759586/178768683-f7aa4f22-e96e-41bf-bde4-4ebb44deea19.png)